src/parser/Parser.cpp

```diff
@@ -226,13 +234,15 @@ void Parser::const_declarations(vector<unique_ptr<ConstantDeclarationNode>> & co
             }
         }
     }
-    // [<VAR>, <TYPE>, <PROCEDURE>, <END>, <BEGIN>]
-    resync({ TokenType::kw_type, TokenType::kw_var, TokenType::kw_procedure, TokenType::kw_begin, TokenType::kw_end });
+    if (consts.size() == 0) {
+        logger_.error(token->start(), "empty CONST section");
```

Comment:
I'm not sure "section" is the correct terminology. The language report simply talks about these as "declarations". A better error message, therefore, would be `empty CONST section`.